### ISSUE TITLE
S3 Exporter in ADOT

### ISSUE THREAD

**Issue by anish-aws** [aws-observability/aws-otel-collector#1615]
**Is your feature request related to a problem? Please describe.**
I am working with a team that wants to export application traces into long term storage. The team has needs to perform custom analysis (things not provided by services like DataDog) over the long term and also analyse traces data in conjunction with other data sets from various sources. They want the ability to move the trace data into cheap and durable storage like S3 and use tools like Athena, EMR, SageMaker, etc to perform analysis.

**Describe the solution you'd like**
The solution requested would be the ability to configure an S3 exporter, similiar to other exporters like X-Ray exporter or Prometheus Exporter. The exporter would take in configuration like the bucket name, path, etc and the ADOT collector will ship trace information into the S3 bucket. From there on downstream systems can consume it as they like. Current use case for the team is to export traces, but future use cases exist for similiar functionality with logs and metrics

**Describe alternatives you've considered**
Other options like Kinesis Exporter and X-Ray exporter was considered. this adds additional step in the data movement to downstream systems and cost. This also requires a substantial engineering effort to standup the solution and maintain it. Analysis inside current services like DataDog or Jaeger is limited as performing long term query analysis with multi source data sets is either technically impossible (such as exporting data from DataDog) or cost prohibitive.
Open-source alternatives like Parquet exporter are currently in beta and would also require the engineering time to build a solution to move data from the container system into S3. The expectation is for the S3 exporter and the collector to do this activity, if built

**Comment by anish-aws** [comment]
Commenting for bot. but also, any update on this request?

**Comment by mhausenblas** [comment]
@anish-aws we're in the review process, gathering input. No confirmed ECD yet.

**Comment by anish-aws** [comment]
Linking open issue from Upstream that seems to have a sponsor as of last week
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/2835

**Comment by kevinslin** [comment]
in case folks were curious - there is now alpha support for s3 exporter in [opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awss3exporter)

**Comment by odeke-em** [comment]
@Aneurysm9 @mhausenblas howdy howdy? I can take on this issue and bring in the exporter that was implemented in https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awss3exporter

**Comment by mhausenblas** [comment]
Thanks for your offer @odeke-em, much appreciated. However, this component is not on our short term roadmap, yet.

**Comment by mariusdrodten** [comment]
> Thanks for your offer @odeke-em, much appreciated. However, this component is not on our short term roadmap, yet.

Hi, 
Is there any update on it?
Or is there a way to use the opentelemetry collector builder to add this exporter to this distribution?

**Comment by milkfrog** [comment]
Is there any date for this release?

**Comment by markl11** [comment]
Yes, please can you give us an approximate release date. Or is ADOT dead? The roadmap looks pretty stale.

**Comment by mhausenblas** [comment]
Thanks for your question @markl11 and concerning:

> can you give us an approximate release date

Once we have a concrete time line, will share.

**Comment by markl11** [comment]
> Thanks for your question [@markl11](https://github.com/markl11) and concerning:
> 
> > can you give us an approximate release date
> 
> Once we have a concrete time line, will share.

Take your time....it's only been 3 years since this feature was requested. Lol. Sorry for the sarcasm, but you should really take it off the roadmap at this point given the lack of progress.

**Comment by dazzle1979** [comment]
Why would you use the aws-otel-collector when you have [opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib)? This project already includes a S3 exporter: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awss3exporter
